### DISCUSSION TITLE
Campaign should not start at 0 difficulty

### DISCUSSION THREAD

**Discussion by the1eyedduck** [FakeFishGames/Barotrauma#5970] [Unstable feedback]
Current campaign start at 0% difficulty in cold caverns and almost nothing spawn. Even in later level, barely anything spawn (most notable is the The Great Void and Hydrothermal Wastes, maybe some bug disable spawning events?) which make traveling between A to B and biomes to biomes feel like a chores.
HOWEVER after complete the campaign and been teleported to cold cavern again (this time start at 25% difficulty), thing get exicting, swarm of crawler patrol the sea, hammerhead every now and then chase the vessel, AND a nature spawn moloch block the way out forcing player to use stealth. (the second start after teleported spawn more than the entire first campaign, again something likely was bugged with the spawn event)
It was exciting, environment feel alive, and most importantly fun.

I would suggest campaign should start at 25% instead of 0% as it would give everyone more thing to do, likely reduce griefing from boredom and most importantly set the theme of the moon as a hostile environment.

i also wanna thank the dev team for the continuously support the game unlike most of other early access game (and most importantly making drone useable with the introduction of enemy pirate ship)

Happy 2 years anniversary of Barotrauma and it dev team!

**Comment by civilCornball** [comment]
The ability to start in later areas would definitely aid public campaigns.

**Comment by Nyerske** [comment]
Or definitely make it optional how difficult players want it, but fully agree. Early game's rather boring but in its defence it servers a purpose for new players. A more player friendly way to introduce people to the game. But for people whose more experienced, a way to edit this would be awesome.

**Comment by Udrakan** [comment]
Moloch causes you to go stealth? Whats it gonna do, shake your screen?

**Reply by the1eyedduck** [reply]
turn off sonar and drive under or above it took no second and save 1-2 piecing box (1 titanium for ~200 money so around ~400 cost minimum per wild moloch kill) + not having to run around and weld hull. Then use that 2 piecing box for monster hunt which profit 3000-6000 for just under 1k cost of ammunition. killing wild moloch waste money, time and resources that can be avoid for barely 2 second extra, but it is exciting to encounter in the wild than just empty water.

**Reply by cshingle** [reply]
I run a public server. When I see a cluster of Molochs I head strait for the middle of the cluster calmly suggesting people get to the guns. The crew is going to be killing something, better monsters than each other.

**Comment by mygamingaccount** [comment]
Why don't you just make 0% a little bit more difficult and have that be the new starting difficulty?
